lib/src/client.dart

```diff
@@ -187,10 +187,14 @@ class SockJSOptions {
   /// can be useful if you need to disable certain fallback transports.
   final List<String> transports;
 
+  /// The minimum timeout, in milliseconds.  If sockJS's internally calculated
+  /// timeout is higher, that will be used instead.
+  final int timeout;
```

Comment:
nit, thoughts?
```suggestion
  final int timeoutMs;
```

Reply:
Sockjs calls it timeout, I decided to follow their api.  But, maybe clarity is better than congruency?

Reply:
Actually,
```suggestion
  final int minTimeoutMs;
```

Reply:
Meh, if they use timeout, let's use timeout.